### DISCUSSION TITLE
Primevue and tailwind

### DISCUSSION THREAD

**Discussion by BrysonFl** [primefaces/community#750] [PrimeVue]
Good day, I tried to use Primevue in my project with tailwind. When I configured the nuxt.config.js the project worked, but when I added the tailwind file the project broke, I don't know what I needed to do to solve this problem.

My tailwind.css file has:
![Screenshot 2023-12-21 at 1 35 32 PM](https://github.com/primefaces/community/assets/29582875/138a18ff-ed52-4225-94ca-5017575304c5)

And, my nuxt configuration file has:
![Screenshot 2023-12-21 at 1 35 51 PM](https://github.com/primefaces/community/assets/29582875/0c1700ae-abdf-4a21-b015-a18aa2bc4ae8)

But my styles are broken
![Screenshot 2023-12-21 at 1 36 39 PM](https://github.com/primefaces/community/assets/29582875/d91637a2-5470-43a4-a10a-505616064355)

Thanks for the answers

**Comment by vanarok** [comment]
I have similar behavior. The project styles break after adding: 
```css

/* tailwind.css */
@layer tailwind-base {
  @tailwind base;
}

@layer tailwind-utilities {
  @tailwind components;
  @tailwind utilities;
}

```
My project uses vite, tailwind

**Reply by hengvisal99** [reply]
Hi do your tailwind and primevue styles work together?

**Comment by vanarok** [comment]
Downgraded the version to 3.34.0, there is no bug on it.

**Reply by TackJordy** [reply]
Struggled a whole day because of this, thanks for the answer!

**Reply by Tyde** [reply]
For newer versions you have to add the primevue layer as documented here: https://primevue.org/csslayer/#tailwind
